### PR TITLE
How to deploy wordpress

### PR DESCRIPTION
**Pull request by MahdiZareie**

### CODE REVIEW

File: blog/articles/how-to-deploy-wordpress.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+
+```
+The image version: v1
+Your service name: mywp
+Your service deployed successfully.
+The service is accessible via following link:
+http://mywp.[YOUR-NAMESPACE].fandogh.cloud
+
+```
+اگر به آدرس بالا مراجعه کنید با صفحه نصب wordpress مواجه میشید، توی صفحه مربوط به تنظیمات دیتابیس شما نیاز دارید که اطلاعات ارتباط با MySQL رو در اختیار وردپرس قرار بدید.
```
Comment (s-soroosh) [inline blog/articles/how-to-deploy-wordpress.md]: This is not the right way to connect to mysql.
Use env variables instead.

File: blog/articles/how-to-deploy-wordpress.md [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+### خلاصه مراحل
+1.  یک اکانت فندق می‌سازید و فندق cli ‌رو نصب کنید
+2. یک سرویس MySQL ‌از طریق دستور managed-service ‌ایجاد می‌کنید.
+3. یک داکر فایل برای وردپرس می‌نویسید و روی فندق پابلیش می‌کنید.
+4. با ست کردن صحیح environment variables وردپرس رو دیپلوی می‌کنید.
+
+::: tip نکته
+اگر حوصله خوندن ندارید می‌تونید 
+ [از این ریپو](https://github.com/fandoghpaas/fandogh-examples/tree/master/wordpress-mysql)
+ ریپو استفاده کنید :-D
```
Comment (s-soroosh) [inline blog/articles/how-to-deploy-wordpress.md]: Extra repo

File: blog/articles/how-to-deploy-wordpress.md [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+:::
+
+
+### قدم اول: ساخت اکانت فندق و نصب CLI
+کافیه یک سر به آدرس [فندق](http://fandogh.cloud/) بزنید و روی دکمه ثبت‌نام کلیک کنید، فکر نمی‌کنم بیشتر از ۱ دقیقه طول بکشه.
```
Comment (s-soroosh) [inline blog/articles/how-to-deploy-wordpress.md:18]: No space before comma or dot

File: blog/articles/how-to-deploy-wordpress.md (hunk 2 above)
Comment (MahdiZareie) [reply blog/articles/how-to-deploy-wordpress.md]: What?

**Pull request merged**